configuration/pipeline_config.json

```diff
@@ -0,0 +1,77 @@
+{
+  "PipelineName": "UNICEF-COVID19-KE",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/unicef-kenya-textit-token.txt",
+      "ContactsFileName": "unicef_covid19_ke_contacts",
+      "ActivationFlowNames": [
+        "unicef_covid19_ke_s01e01_activation",
+        "unicef_covid19_ke_s01e02_activation",
+        "unicef_covid19_ke_s01e03_activation",
+        "unicef_covid19_ke_s01e04_activation",
+        "unicef_covid19_ke_s01e05_activation"
+      ],
+      "SurveyFlowNames": [
+        "unicef_covid19_ke_s01_demog"
+      ],
+      "TestContactUUIDs": [
+        "94075043-57c6-43c7-8b89-880478ccad8e",
+        "1b895d33-a592-472f-9d01-dbf9b412a6e1"
+      ]
+    }
+  ],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "UNICEF_KENYA_phone_number_avf_phone_id"
```

Comment:
Is this supposed to be different from the project name?

Reply:
Have used the textit instance name with a forwarding thinking design of if we get another UNICEF Kenya project we building up from this table.